### PR TITLE
#6 Task for automating publication of a jar with the Tableau-dependen…

### PR DESCRIPTION
**Pull request by rjrudin**
…t source files

### CODE REVIEW

**Comment by rjrudin** [comment]
@vivekmuniyandi Please review this as well.

File: build.gradle
Comment (vivekmuniyandi) [inline build.gradle]: We forgot to remove these.

**Review by vivekmuniyandi — APPROVED** [review]
This looks good! It can be merged after we remove the following lines from build.gradle

compile files("lib/tableaucommon.jar")
compile files("lib/tableauextract.jar")

**Comment by rjrudin** [comment]
Thanks @vivekmuniyandi for catching that, I just removed those two lines.

Are you able to merge the PR? Wasn't sure if you had permission to do so yet, was hoping you do.

**Comment by vivekmuniyandi** [comment]
@rjrudin I am not able to merge. Looks like I don't have write access to this repository. I will send out a mail to Github owners. Thanks.

**Pull request merged**
